src/game_manager.py

```diff
@@ -194,29 +194,30 @@ def _clear_cell_value(self, cell: SudokuCell):
     def _fill_cell(self, cell: SudokuCell, number: str, ctrl_is_pressed=False):
         UIHelpers.clear_conflicts(self.conflict_cells)
         row, col = cell.row, cell.col
+        valid_numbers = "123456789"
+        if number in valid_numbers:
```

Comment:
```suggestion
        if number != 0:
```
This is shorter and more understandable.

Reply:
hi @sepehr-rs i fix this